src/main/java/com/sap/oss/phosphor/fosstars/tool/report/OssRulesOfPlayGitHubIssuesReporter.java

```diff
@@ -86,8 +87,13 @@ public void createIssuesFor(GitHubProject project) throws IOException {
       String issueHeader = printTitle(violation);
       List<GHIssue> existingGitHubIssues = fetcher.gitHubIssuesFor(project, issueHeader);
       if (existingGitHubIssues.isEmpty()) {
-        fetcher.createGitHubIssue(project, printTitle(violation), printBody(violation));
-        LOGGER.info("New issue: " + issueHeader);
+        try {
+          fetcher.createGitHubIssue(project, printTitle(violation), printBody(violation));
+          LOGGER.info("New issue: " + issueHeader);
+        } catch (GHFileNotFoundException issuesDisabledException) {
+          LOGGER.warn("Creating issues is disabled for project {}", project.toString());
+          break;
+        }
```

Comment:
I am not sure that `GHFileNotFoundException` means only that issues are disabled.

To check whether issues are enabled or not, you can use `hasIssues()` method

https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHRepository.html#hasIssues()
https://docs.github.com/en/rest/reference/repos#create-an-organization-repository--parameters

Then, you don't need to break the loop if an exception occurred due to some other problems.

```suggestion
        } catch (GHFileNotFoundException e) {
          LOGGER.warn("Could not create an issue for project {}", project.toString(), e);
        }
```

Reply:
Of course, I'm not 100% sure about the mapping of the response codes with the exceptions as I haven't digged deeper into the code. However, according to the response codes I'd say that this exception might only occur in case of a 404 or 410 (the latter [occurs if creating issues is disabled](https://docs.github.com/en/rest/reference/issues#create-an-issue) so I was confident enough to use it here.

Moreover, due to the rate limitation, I tried to avoid every GH API possible - that's why I used the try/catch approach instead of the `hasIssues()` method. But probably getting the repository information doesn't count against the issue creation numbers and is probably cached anyway, so the effect is minimal. Therefore, I don't object changing it accordingly. Expect a new commit shortly. :)